coda-prototype/initUI.js

```diff
@@ -394,6 +394,25 @@ function initUI(dataset) {
     var messagePanel = $("#message-panel");
     var editorRow = $("#editor-row");
 
+    $.getJSON("version.json", version => {
+        let hash = version.hash === "develop" ? "develop" : `${version.hash.substring(0, 6).toUpperCase()}`;
```

Comment:
As discussed let's refactor this to unify ln398 and ln400

Reply:
done